src/index.html

```diff
@@ -11,12 +11,101 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          alt="logo"
```

Comment:
alt attribute should describe the image content
<img width="479" alt="image" src="https://github.com/user-attachments/assets/f389364d-771a-4f12-b1d3-0ed86e0553d7">

Reply:
I've corrected errors and changed part 

<a href="index.html">
        <img
          src="images/logo.png"
          alt="Moyo Logo"
          class="logo__img"
        />